lib/constants/category.ts

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line typescript-enum/no-enum
+export enum Category {
+  Ansible = 'ansible',
+  Batect = 'batect',
+  Bazel = 'bazel',
+  CD = 'cd',
+  CI = 'ci',
+  C = 'c',
+  Kubernetes = 'kubernetes',
+  Ruby = 'ruby',
+  Dotnet = 'dotnet',
+  Rust = 'rust',
+  Swift = 'swift',
+  PHP = 'php',
+  JVM = 'jvm',
```

Comment:
Not Java?

Reply:
I have chosen `jvm` over `java` as multiple languages use the same packages managers. e.g. Kotlin, Groovy, Scalar and so on.